deploy.sh

```diff
@@ -97,9 +97,6 @@ else
   aws s3 cp ~/.ivy2/local ${PUBLISH_S3_URL}/${GITHUB_TAG_1}/ --recursive
 fi
 
-echo "Run White Source scan"
-whitesource/run_whitesource.sh
```

Comment:
There are another 2 places to call `run_whitesource.sh`, they should be removed too.

Reply:
good catch, thanks! just pushed an update